analyzer_naive.cc

```diff
@@ -0,0 +1,41 @@
+#include <chrono>
+#include <memory>
+
+#include "input.h"
+#include "output.h"
+#include "rawtodigi_naive.h"
+
+namespace {
+  constexpr int NLOOPS = 100;
+}
+
+namespace naive {
+  void analyze(Input const& input, std::unique_ptr<Output>& output, double& totaltime) {
+    for (int i = 0; i < NLOOPS; ++i) {
+      output = std::make_unique<Output>();
```

Comment:
@fwyzard I reminded myself that resetting the `output` for each iteration is needed to clear the `GPUSimpleVector err` (ok, there are other ways but this way resembles the `Event` in CMSSW).